### PR TITLE
Update bash scripts

### PR DESCRIPTION
**Pull request by wolf31o2**
This resolves issues from running CDAP from a directory that isn't the root of the SDK.

### CODE REVIEW

File: cdap-common/bin/functions.sh [hunk 1]
```diff
@@ -748,6 +748,9 @@ cdap_sdk_start() {
   CLASSPATH=$(find "${CDAP_HOME}/lib" -type f | sort | tr '\n' ':')
   CLASSPATH="${CLASSPATH}:${CDAP_HOME}/conf/"
 
+  # SDK requires us to be in CDAP_HOME
+  cd $(cdap_home)
```
Comment (awholegunch) [inline cdap-common/bin/functions.sh]: Should this not be: (a little confusing with both the variable and the method name similar)

```
cd ${CDAP_HOME}
```

Comment (wolf31o2) [reply cdap-common/bin/functions.sh]: Either one works.

Comment (wolf31o2) [reply cdap-common/bin/functions.sh]: I switched to using the variable.

Comment (awholegunch) [reply cdap-common/bin/functions.sh]: OK. That makes it less confusing, as it otherwise suggests that the variable was no longer valid.

**Comment by awholegunch** [comment]
AFAICT, LGTM.

**Comment by wolf31o2** [comment]
Just for posterity:

``` bash
$ pwd
/tmp
$ ls -l cdap
lrwxrwxrwx 1 chris chris 39 Sep 22 17:42 cdap -> /home/chris/tmp/cdap-sdk-4.0.0-SNAPSHOT/bin/cdap
$ ./cdap sdk status
CDAP Standalone (SDK) running as PID 29255
$ ~chris/tmp/cdap-sdk-4.0.0-SNAPSHOT/bin/cdap sdk status
CDAP Standalone (SDK) running as PID 29255
$ cd ~chris/tmp/cdap-sdk-4.0.0-SNAPSHOT/bin; ./cdap sdk status; cd -
CDAP Standalone (SDK) running as PID 29255
/tmp
```

**Pull request merged**